docs/Solidity.g4

```diff
@@ -385,6 +385,8 @@ NumberUnit
   : 'wei' | 'szabo' | 'finney' | 'ether'
   | 'seconds' | 'minutes' | 'hours' | 'days' | 'weeks' | 'years' ;
 
+Gwei: 'gwei' ;
```

Comment:
Including `Gwei` in `NumberUnit` does not work?

Reply:
Unfortunately not. The Lexer turns it into a NumberUnit token which and the 'gwei' can then no longer be matched for `identifier`

Reply:
Sorry to bother: Even if you put `Gwei` above `NumberUnit`?

Reply:
No success:
```
[1/4] Testing /home/marenz/eth/solidity/test/libsolidity/syntaxTests/denominations/denominations.sol FAILED
line 5:21 missing ';' at 'gwei'
line 5:26 mismatched input '/' expecting {'from', 'override', 'calldata', 'address', 'gwei', 'constant', 'immutable', 'internal', 'private', 'public', Identifier}
 ✘ marenz@edox  ~/eth/solidity   issue-8716 ●  cat
^C
 ✘ marenz@edox  ~/eth/solidity   issue-8716 ●  cat /home/marenz/eth/solidity/test/libsolidity/syntaxTests/denominations/denominations.sol
contract C {
	uint constant a = 1 wei + 2 szabo + 3 finney + 4 ether;
	uint constant b = 1 seconds + 2 minutes + 3 hours + 4 days + 5 weeks;
	uint constant c = 2 szabo / 1 seconds + 3 finney * 3 hours;
	uint constant d = 2 gwei / 1 seconds + 3 finney * 3 hours;
}
// ----
```